composer.json

```diff
@@ -74,10 +74,10 @@
   "require-dev": {
     "composer/composer": "^1.10.0",
     "phpunit/phpunit": "^8",
-    "fzaninotto/faker": "~1.9.0",
+    "fzaninotto/faker": "dev-master",
```

Comment:
What about changing it to https://github.com/FakerPHP/Faker It has a php 8 release

Reply:
Yeah, was already done, but then my internet broke and forget to push after it was fixed ^^ .

Reply:
Can we use the commit id here? It would protect us from unintended updates or manipulation.

Reply:
I just saw that the current pr is using a version number :+1: